github-pages.yml

```diff
@@ -8,19 +8,27 @@ site:
 
 content:
   sources:
-    - url: ./
+    - url: .
+      branches: HEAD
       start_path: documentation
       
 asciidoc:
   attributes:
     release-version: master
+    # Set myrepo to the organization in the image registry you want to reference in the tutorial
+    myrepo: your-repo
+    # Set the docker-host attribute to the hostname that should be used to refer to target of curl commands in a container
+    # for example: docker.for.mac.localhost
+    docker-host: localhost
+    experimental: true
+    curl-loop-sleep-time: .8
   extensions:
     - ./lib/tab-block.js
     - ./lib/remote-include-processor.js
     
 ui:
   bundle:
-    url: https://github.com/redhat-developer-demos/rhd-tutorial-ui/releases/download/v0.1.9/ui-bundle.zip
+    url: https://github.com/hatmarch/course-ui/releases/download/v0.1.12/ui-bundle.zip
```

Comment:
Is there any problem with sending a PR for this to rhd-tutorial so everything is tracked in the same repo?

Reply:
@lordofthejars This has been addressed in my recent changes to this pull request